### ISSUE TITLE
Suicide?

### ISSUE THREAD

**Issue by jonstokes** [jonstokes/shootercontrol#15]
If someone in your shooter network commits suicide, it seems like that shouldn't be a disbanding offense, even though it's technically a gun crime, I guess. I want to flag this for further discussion and possibly include something about it in the proposal.

**Comment by tsgsjeremy** [comment]
Yeah I keep going back and forth on the punishment issue. On the one hand it _feels_ like the punishments you proposed are too harsh, but on the other hand they also _seem_ correct. I have a general concept of a spectrum in mind but it hasn't yet coalesced into a specific and coherent set of ideas.
